src/libtrx/include/libtrx/game/anims/types.h

```diff
@@ -15,6 +16,22 @@ typedef struct {
     int16_t link_frame_num;
 } ANIM_RANGE;
 
+typedef struct {
+    ANIM_COMMAND_TYPE type;
+    void *data;
+} ANIM_COMMAND;
+
+typedef struct {
+    int16_t fall_speed;
+    int16_t speed;
+} COMMAND_VELOCITY_DATA;
+
+typedef struct {
+    int16_t frame_num;
+    int16_t effect_num;
+    ANIM_COMMAND_ENVIRONMENT environment;
+} COMMAND_EFFECT_DATA;
```

Comment:
Let's prefix with `ANIM_` this and the previous struct.

Reply:
Done